extract.py

```diff
@@ -262,7 +262,7 @@ def write_metadata(n_rows, creation_time):
     timestamp = datetime.strftime(creation_time, "%Y%m%dT%H%M%S")
     metaname = f"temp-data/metadata-{timestamp}.json"
```

Comment:
This PR needs to be tested against Windows as this will likely break on Windows.

This error was introduced in the [previous metadata PR](https://github.com/mitre/data-owner-tools/blame/cf550cafc40f2352156dcca6619d2094db0b6d37/extract.py#L262-L263), but we should fix it now.

Reply:
See now commits, code is fixed and tested on windows